h2o-py/h2o/estimators/infogram.py

```diff
@@ -837,6 +922,23 @@ def data_fraction(self):
         and less than or equal to 1.0 is acceptable.
 
         Type: ``float``, defaults to ``1.0``.
+
+        :examples:
+
+        >>> import h2o
+        >>> from h2o.estimators.infogram import H2OInfogram
+        >>> h2o.init()
+        >>> f = "https://erin-data.s3.amazonaws.com/admissible/data/taiwan_credit_card_uci.csv"
+        >>> col_types = {'SEX': "enum", 'MARRIAGE': "enum", 'default_payment_next_month': "enum"}
+        >>> df = h2o.import_file(path=f, col_types=col_types)
+        >>> train, test = df.split_frame(seed=1)
+        >>> y = "default_payment_next_month"
+        >>> x = train.columns
+        >>> x.remove(y)
+        >>> pcols = ["SEX", "MARRIAGE", "AGE"]
+        >>> ig = H2OInfogram(protected_columns=pcols, data_fraction=0.7)
```

Comment:
Does this work the same as `split_frame`?

Reply:
cc: @hannah-tillman